packages/wallet/src/types/ObservableWallet.ts

```diff
@@ -103,3 +106,7 @@ export interface ObservableWallet<Tip = WC.Tip> {
   submitTx(tx: Cardano.NewTxAlonzo): Promise<void>;
   shutdown(): void;
 }
+
+export interface ObservableWallet extends ObservableWalletCore {
+  readonly tip$: Observable<Cardano.Tip>;
```

Comment:
I personally liked the generics approach better. Then, one can create types that have the generics already filled in. But it could be just personal preference 😄 
`export interface ObservableWalletCore<Tip = WC.Tip> {...}`
`export type ObservableWallet = ObservableWalletCore<Cardano.Tip>`